resources/js/components/FormTabs.vue

```diff
@@ -89,15 +90,40 @@ export default {
   },
   watch: {
     errors: {
-      handler: function(errors) {
-        let vm = this;
+      handler(errors) {
+        const errorFields = Object.keys(errors.errors);
+        const errorTabs = [];
         let goToTab = false;
-        Object.keys(vm.tabs).forEach(function(key) {
-          if (vm.tabHasErrors(vm.tabs[key]) && !goToTab) {
-            goToTab = true;
-            return vm.handleTabClick(vm.tabs[key]);
+        // If there were validation errors
+        if(errorFields.length > 0) {
+          // check which tabs have an error
+          _.forEach(this.field.fields, field => {
+            if (typeof field.tab === 'undefined') return;
+
+            if (errorFields.indexOf(field.attribute) >= 0 && errorTabs.indexOf(field.tab) === -1) {
+              errorTabs.push(field.tab);
+            }
+          });
+        }
+
+        // If there were tabs with errors
+        if(errorTabs.length > 0) {
+          // Go to the first tab that errors
+          Object.keys(this.tabs).forEach(tab => {
```

Comment:
You're unnecessarily looping through each entry here, pretty sure this can be replaced with a `find`.

```js
Object.keys(this.tabs).find(tab => errorTabs.indexOf(tab) >= 0);

if (tab) {
    this.handleTabClick(this.tabs[goToTab]);
}
```